ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -158,6 +171,7 @@ export class EvalCommand extends Command<EvalCommandReturn> {
 
     const mergedResults = _([pipe, result])
       .flatMap((current) => _.values(current))
+      .value()
```

Comment:
I remember that you had a linting issue here but it's an unnecessary change that adds more complexity to the operation and is irrelevant to this PR. Let's omit this for now 🙏🏻

Reply:
done